app/serializers/export/post_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Export
+  class PostSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
+               :public,
+               :diaspora_handle,
+               :type,
+               :image_url,
+               :image_height,
+               :image_width,
+               :likes_count,
+               :comments_count,
+               :reshares_count,
+               :favorite
```

Comment:
Eh, I don't remember that one at all, must be from the time I didn't watch the project. It was introduced in 75d25e947502851e2b79a6f9a101a2a83d2eb363